source/replication/server-side-replication-resynchronize-remote.rst

```diff
@@ -0,0 +1,546 @@
+.. _minio-bucket-replication-resynchronize:
+
+
+========================================
+Resynchronize Bucket from Remote Replica
+========================================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 1
+
+The procedure on this page resynchronizes the contents of a MinIO
+bucket using a healthy replication remote. Resynchronization supports
+recovery after partial or total loss of data on a MinIO deployment in a 
+replica configuration.
+
+For example, consider a MinIO active-active replication configuration similar
+to the following:
+
+.. image:: /images/replication/active-active-twoway-replication.svg
+   :width: 600px
+   :alt: Active-Active Replication synchronizes data between two remote deployments.
+   :align: center
+
+Resynchronization allows using the healthy data on one of the participating
+MinIO deployments as the source for rebuilding the other deployment.
+
+Resynchronization is a per-bucket process. You must repeat resynchronization
```

Comment:
there is no restriction to disable replication resync as of now when site replication is configured

Reply:
I think the assumption is configuring site replication prevents configuring bucket replication and vice versa.

I am planning on letting Daryl tackle site replication, so this will be good for him to test and see.